Windows-Installation_Instructions.txt

```diff
@@ -0,0 +1,51 @@
+INSTALLATION OF SIMPHONY, SIMPHONY-NCAD AND NFLUID
+==================================================
+
+1. Install Python 2.7
+   - https://www.python.org/downloads/
+     (Windows package python-2.7.11.msi)
+
+2. Configure Python to use MinGW
+   - create 'distutils.cfg' file under 'C:\Python27\Lib\distutils' folder with the text:
+[build]
+compiler = mingw32
```

Comment:
The `libNCad.dll` library —not publicly distributed, but available to the Consortium partners, as explained by @gromansg above— is compiled using MinGW (actually, with the Windows package `mingw-get-inst-20111118.exe`, suite corresponding to gcc and g++ version 4.6.1, to be more precise).